docs/decisions/0001-coodinator-design.rst

```diff
@@ -0,0 +1,81 @@
+#######################
+0001 Coordinator Design
+#######################
+
+Status
+******
+
+**Draft**
+
+Context
+*******
+
+edX's monolithic Ecommerce Service was meant to be a full-featured product catalog, discounting, and order management service and has grown in complexity for many years until it has become very hard to maintain, let alone evolve.  In that time, more and better hosted services have become available for solving many of these problems, and complex use cases have evolved that make flexibility for Open edX operators vital.
+
+
+Decisions
+*********
+
+- This application will be a Django-based independently deployed application with a MySQL backend database and redis cache.
+- This application will work as a modular coordinator between edX platform services and various other hosted or home grown solutions that provide functionality relevant to commercial transactions.
+- This application will not provide a customer-facing user interface.
+- This application will not maintain state for business objects.
+
+
+Consequences
+************
+
+Django based
+============
+
+
+
+Highly modular
+==============
+
+The world of online commerce is rapidly changing, and with so many moving pieces it is important to Open edX operators to be able to use the services and tools that are the best fit for them, in the ways that they need to. As such we have decided to use a system of decoupled events to allow maximum flexibility in implementation. The exact details of the event mechanisms will be addressed in subsequent ADRs.
+
+The system will have a fixed set of default messages handling most current use cases (ex: Payment Received, Refund Requested) and a built in set of message handlers that can be enabled via configuration which interact with other Open edX systems (LMS notified upon Payment Received). Additional Django Apps can be installed in the usual fashion which to offer additional handlers.
+
+In addition to handlers, the system will provide default URL endpoints to receive incoming API calls / callbacks from external services, and allow extensibility via the usual Django app mechanisms to add new API endpoints and callback handlers which can, in turn, fire off custom messages.
```

Comment:
This makes it sound like the core will vend URLs / APIs, again that is not my expectation.